docs/pages/installation/minikube.md

```diff
@@ -59,16 +59,26 @@ Perform the following steps in order:
 
 ### 2. Install Meshery
 
-Follow the [installation steps](/guides/mesheryctl) to setup the mesheryctl CLI and install Meshery.
+Follow the [installation steps](/installation/quick-start) to setup the mesheryctl CLI and install Meshery.
 
-**Note**: If you are using docker driver, after completing meshery installation steps execute below command to establish connectivity between Meshery and Kubernetes server :
+**Users using docker driver**: 
+
```

Comment:
nit remove extra line

Reply:
@subhamkrai we will also want to change to "...after completing **the** Meshery installation, execute the following command to..."

Use "cluster", not "server" to refer to Kubernetes. Capitalize M in Meshery.